src/routes/auth.route.js

```diff
@@ -38,7 +40,35 @@ const authRouter = Router();
  *       5XX:
  *        description: Unexpected error.
  */
-authRouter.post('/signup', validateNewUser, signup);
+authRouter.post('/signup', validateReqBody(signupSchema), signup);
+
+/**
+ * @swagger
+ *
+ * /api/v1/users:
+ *   post:
+ *     description: User Account Confirmation
```

Comment:
You may also want to add a `- tags:` attribute here, to enable proper naming for this route in the documentation route.

Reply:
Done. thanks